daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2471,6 +2472,7 @@ func (devices *DeviceSet) Status() *Status {
 		}
 	}
 
+	status.MinFreeSpace = fmt.Sprintf("%v%%", devices.minFreeSpacePercent)
```

Comment:
I am no the fence too. Displaying both percentage and absolute values are useful for human beings but if software is using this output, they can easily calculate one way or other.

At the same time, displaying both seems little redundant so I would rather avoid that and display either percentage or absolute value and let user and software calculate other if need be.

Given other values are absolute in docker info output, I think displaying absolute values for minimum free space is reasonable too. This will be fine if and when we support absolute values for dm.min_free_space parameter.

In summary, I think displaying either percentage or absolute both should be fine. But I am willing to implement absolute values for the sake of uniformity with other values.